clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java

```diff
@@ -21,80 +21,69 @@
    * @param fileName the name of the file to be created
-   * @param tachyonStorageType {@link TachyonStorageType} used to create the file
-   * @param underStorageType {@link UnderStorageType} used to create the file
+   * @param writeType {@link WriteType} used to create the file
    * @param len file size
-   * @return the {@link TachyonFile} of the created file
    * @throws IOException if {@code path} is invalid (e.g., illegal URI)
    */
-  public static TachyonFile createByteFile(TachyonFileSystem tfs, String fileName,
-      TachyonStorageType tachyonStorageType, UnderStorageType underStorageType, int len)
-      throws IOException {
-    return createByteFile(tfs, new TachyonURI(fileName), tachyonStorageType, underStorageType, len);
+  public static void createByteFile(FileSystem tfs, String fileName,
+      WriteType writeType, int len) throws IOException {
+    createByteFile(tfs, new TachyonURI(fileName), writeType, len);
   }
 
   /**
    * Creates a simple file with {@code len} bytes.
    *
-   * @param tfs a {@link TachyonFileSystem} handler
+   * @param tfs a {@link tachyon.client.file.FileSystem} handler
    * @param fileURI URI of the file
-   * @param tachyonStorageType {@link TachyonStorageType} used to create the file
-   * @param underStorageType {@link UnderStorageType} used to create the file
+   * @param writeType {@link WriteType} used to create the file
    * @param len file size
-   * @return the {@link TachyonFile} of the created file
    * @throws IOException if {@code path} is invalid (e.g., illegal URI)
    */
-  public static TachyonFile createByteFile(TachyonFileSystem tfs, TachyonURI fileURI,
-      TachyonStorageType tachyonStorageType, UnderStorageType underStorageType, int len)
-      throws IOException {
-    OutStreamOptions options = new OutStreamOptions.Builder(ClientContext.getConf())
-        .setTachyonStorageType(tachyonStorageType).setUnderStorageType(underStorageType).build();
-    return createByteFile(tfs, fileURI, options, len);
+  public static void createByteFile(FileSystem tfs, TachyonURI fileURI,
+      WriteType writeType, int len) throws IOException {
+    CreateFileOptions options = CreateFileOptions.defaults().setWriteType(writeType);
+    createByteFile(tfs, fileURI, options, len);
   }
 
   /**
    * Creates a simple file with {@code len} bytes.
    *
-   * @param tfs a {@link TachyonFileSystem} handler
+   * @param tfs a {@link tachyon.client.file.FileSystem} handler
    * @param fileURI URI of the file
    * @param options client options to create the file with
    * @param len file size
-   * @return the {@link TachyonFile} of the created file
    * @throws IOException if {@code path} is invalid (e.g., illegal URI)
    */
-  public static TachyonFile createByteFile(TachyonFileSystem tfs, TachyonURI fileURI,
-      OutStreamOptions options, int len) throws IOException {
+  public static void createByteFile(FileSystem tfs, TachyonURI fileURI,
```

Comment:
Options is sort of like an extension point to add new parameters without changing the signature, so it feel like it belongs at the end, similar to varargs or the FileSystem API